README.md

```diff
@@ -131,84 +51,86 @@ DefaultCkbApi ckbApi = new DefaultCkbApi(ckbUrl, mercuryUrl, indexerUrl, false);
+
+### Build transaction with Mercury
+
+[Mercury](https://github.com/nervosnetwork/mercury) is an application for better interaction with CKB chain, providing
+many useful [RPC APIs](https://github.com/nervosnetwork/mercury/blob/main/core/rpc/README.md) for development like
+querying transaction or getting UDT asset information. You need to deploy your own mercury server and sync data with the
+latest network before using it.
+
+Mercury is another way to build transaction. With the help of Mercury, you can build a transaction by simply calling a
+JSON-RPC API. Here we show how to build a CKB transfer transaction with mercury.
+
+```java
+String sender = "ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsq0yvcdtsu5wcr2jldtl72fhkruf0w5vymsp6rk9r";
+String receiver = "ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqvglkprurm00l7hrs3rfqmmzyy3ll7djdsujdm6z";
+
+long ckbAmount = AmountUtils.ckbToShannon(100);   // Convert CKB to Shannon (1 CKB = 10^8 Shannon)
+SimpleTransferPayloadBuilder builder = new SimpleTransferPayloadBuilder();
+builder.addFrom(sender);
+builder.addTo(receiver, ckbAmount);
+builder.assetInfo(AssetInfo.newCkbAsset());
+
+// Get an unsigned raw transaction with the help of Mercury
+TransactionWithScriptGroups txWithScriptGroups = api.buildSimpleTransferTransaction(builder.build());
+```
+
+For more use cases of mercury, please refer to [these test cases](./ckb-mercury-sdk/src/test/java/mercury)
+and [Mercury JSON-RPC documentation](https://github.com/nervosnetwork/mercury/blob/dev-0.4/core/rpc/README.md).
 
 ### Sign and send transaction
+
 To send transaction you build to CKB network, you need to
+
 1. sign transaction with your private key.
 2. send signed transaction to CKB node, and wait it to be confirmed.
 
 ```java
-// Assume that you already have an unsigned transaction named `rawTx`
-Secp256k1SighashAllBuilder signBuilder = new Secp256k1SighashAllBuilder(rawTx);
-
-// Prepare private key to sign
-List<ScriptGroupWithPrivateKeys> scriptGroupWithPrivateKeysList = new ArrayList<>();
-scriptGroupWithPrivateKeysList.add(
-    new ScriptGroupWithPrivateKeys(
-        new ScriptGroup(Arrays.asList(0, 1, 2)),  // indices of input cells that are to be unlocked by this private key
-        Arrays.asList("e79f3207ea4980b7fed79956d5934249ceac4751a4fae01a0f7c4a96884bc4e3")  // your private key
-    )
-);
-
-// Sign transaction with private key
-for (ScriptGroupWithPrivateKeys scriptGroupWithPrivateKeys : scriptGroupWithPrivateKeysList) {
-    signBuilder.sign(scriptGroupWithPrivateKeys.scriptGroup, 
-        scriptGroupWithPrivateKeys.privateKeys.get(0));
-}
-
-// Send transaction
-Transaction tx = signBuilder.buildTx();
-ckbApi.sendTransaction(tx);
+// Before signing and sending transaction, you need to prepare a raw transaction represented by a instance of class `TransactionWithScriptGroups`
+// You can get it by `merucyrApi` or construct it by mamual.
```

Comment:
```suggestion
// You can get it by `merucyrApi` or construct it by manual.
```

I think better to refer previous section `Build transaction with mercury or ckb-indexer`